src/igr-tests/add-rm-dep/run-test.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/sh
 
-../../dinit -d sd -u -p socket -q main &
+if [ $IS_MESON ]; then
+   cd $(dirname $0)
+   DINIT_EXEC=$APPS_PATH/dinit
```

Comment:
How about just set DINIT_EXEC and DINITCTL_EXEC from outside the script (from the build system, makefile and meson?)

Reply:
I fix this in meson builds but I don't have idea about how can I import some variables to igr-tests in make builds?

Reply:
Make commands are just shell commands...
```
check-igr: igr-runner
	DINIT_EXEC=WHATEVER DINITCTL_EXEC=WHATEVER ./igr-runner
```